packages/core/src/sso/index.ts

```diff
@@ -37,6 +37,7 @@ export type SingleSignOnFactory<T extends SsoProviderName> = {
   logo: string;
   logoDark: string;
   description: I18nPhrases;
+  name: I18nPhrases;
```

Comment:
better add a comment for this new field. To distinguish with existing providerName

Reply:
Comments added.